Integrations/src/test/java/io/deephaven/integrations/learn/IndexSetTest.java

```diff
@@ -0,0 +1,81 @@
+package io.deephaven.integrations.learn;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+import java.util.NoSuchElementException;
+import java.util.PrimitiveIterator;
+
+public class IndexSetTest {
+
+    static IndexSet indexSet;
```

Comment:
Generally, bad practice to have these state variables in tests.  Any reason for this to exist?